7.1/Dockerfile

```diff
@@ -8,8 +8,8 @@ RUN sudo apt-get update; \
 # ref: https://github.com/docker-library/php/pull/542
   sudo rm /etc/apt/preferences.d/no-debian-php; \
   sudo apt-get update; \
-  sudo apt-get install mysql-client php7.0-mysql zlib1g-dev; \
```

Comment:
Can we really remove `php7.0-mysql` bindings package?
In MP3 we use PDO, but WP IIRC tries to either MySQL and MySQLi.

Reply:
I had an error during the build and the package was not installed and all tests were passing so I decided to remove that.
We also install mysqli in next step using `docker-php-ext-install`

I ran `docker run mailpoet/wordpress:7.1_20181009.1 php -i` and I can see in output that mysqli is installed so I guess it is ok.